axelrod/_strategy_utils.py

```diff
@@ -47,11 +48,23 @@ def limited_simulate_play(player_1, player_2, h1):
     update_history(player_1, h1)
     update_history(player_2, h2)
 
+
 def simulate_match(player_1, player_2, strategy, rounds=10):
     """Simulates a number of matches."""
     for match in range(rounds):
         limited_simulate_play(player_1, player_2, strategy)
 
+
+def calculate_scores(p1, p2, game):
```

Comment:
I think players could be better actually. Thinking that it might be more future proof? For whatever reason, perhaps at some point we need more than just the histories to calculate the scores? (I have absolutely no idea what that would be but just a thought).